src/vertexfilter.cpp

```diff
@@ -414,25 +414,25 @@ static void decodeFilterOctSimd(signed char* data, size_t count)
 		// unpack z; note that z is unsigned so we technically don't need to sign extend it
 		int32x4_t zf = vshrq_n_s32(vshlq_n_s32(n4, 8), 24);
 
-		// convert x and y to floats and reconstruct z; this assumes zf encodes 1.f at the same bit count
-		float32x4_t x = vcvtq_f32_s32(xf);
-		float32x4_t y = vcvtq_f32_s32(yf);
-		float32x4_t z = vsubq_f32(vcvtq_f32_s32(zf), vaddq_f32(vabsq_f32(x), vabsq_f32(y)));
+		// convert x and y to real_ts and reconstruct z; this assumes zf encodes 1.f at the same bit count
+		real_t32x4_t x = vcvtq_f32_s32(xf);
```

Comment:
These SIMD types would have to have their own typedef, maybe `realx4_t` -- if this sort of SIMD allows doubles? also I see functions that explicitly work on f32

or... can we just disable some features in 64-bit mode if we don't actually use them in our use case?

Reply:
I will just revert those.
Anyway they don't get built, otherwise we'd have had build failures.